### PR TITLE
txm: rename NewTx => TxRequest

### PR DESCRIPTION
**Pull request by aalu1418**
ticket: https://smartcontract-it.atlassian.net/browse/BCI-1209

### CODE REVIEW

**Review by krehermann — DISMISSED** [review]

File: common/txmgr/types/tx_store.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ type TxStore[
 	CheckTxQueueCapacity(fromAddress ADDR, maxQueuedTransactions uint64, chainID CHAIN_ID, qopts ...pg.QOpt) (err error)
 	CountUnconfirmedTransactions(fromAddress ADDR, chainID CHAIN_ID, qopts ...pg.QOpt) (count uint32, err error)
 	CountUnstartedTransactions(fromAddress ADDR, chainID CHAIN_ID, qopts ...pg.QOpt) (count uint32, err error)
-	CreateTransaction(newTx NewTx[ADDR, TX_HASH], chainID CHAIN_ID, qopts ...pg.QOpt) (tx Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], err error)
+	CreateTransaction(newTx TxRequest[ADDR, TX_HASH], chainID CHAIN_ID, qopts ...pg.QOpt) (tx Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], err error)
```
Comment (prashantkumar1982) [inline common/txmgr/types/tx_store.go:39]: nit: txRequest

File: core/chains/evm/txmgr/evm_tx_store.go [hunk 2]
```diff
@@ -1447,7 +1447,7 @@ func (o *evmTxStore) CheckTxQueueCapacity(fromAddress common.Address, maxQueuedT
 	return
 }
 
-func (o *evmTxStore) CreateTransaction(newTx EvmNewTx, chainID *big.Int, qopts ...pg.QOpt) (tx EvmTx, err error) {
+func (o *evmTxStore) CreateTransaction(newTx EvmTxRequest, chainID *big.Int, qopts ...pg.QOpt) (tx EvmTx, err error) {
```
Comment (prashantkumar1982) [inline core/chains/evm/txmgr/evm_tx_store.go:1450]: nit: rename param name to evmTxRequest

File: core/chains/evm/txmgr/evm_tx_store_test.go [hunk 3]
```diff
@@ -1567,7 +1567,7 @@ func TestORM_CreateTransaction(t *testing.T) {
 
 	t.Run("doesn't insert eth_tx if a matching tx already exists for that pipeline_task_run_id", func(t *testing.T) {
 		id := uuid.New()
-		newTx := txmgr.EvmNewTx{
+		newTx := txmgr.EvmTxRequest{
```
Comment (prashantkumar1982) [inline core/chains/evm/txmgr/evm_tx_store_test.go:1570]: nit: Hmm, might make sense to do String search in entire code to replace newTx with txRequest?

**Review by jmank88 — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]
Yay!

**Review by jmank88 — APPROVED** [review]

**Pull request merged**